backend/account/views.py

```diff
@@ -1,3 +1,28 @@
+from django.contrib.auth.models import User
 from django.shortcuts import render
+from .serializers import UserRegisterSerializer
+from rest_framework import generics, status
+from rest_framework.response import Response
+from rest_framework_simplejwt.tokens import RefreshToken
 
-# Create your views here.
+class Register(generics.CreateAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserRegisterSerializer
+
+    def create(self, request, *args, **kwargs):
+        serializer = UserRegisterSerializer(data=request.data)
+        if serializer.is_valid():
+            user = serializer.create()
+            tokens = self.get_tokens_for_user(user)
+
+            return Response({'success': True,'tokens': tokens})
+            
+        return Response({'success': False})   
```

Comment:
yo hamle diuso sangai garya theem tara better chai,
remove success:true or false no needed just directly return refresh and access same response as login...
And in case of error, try this;
`raise serializers.ValidationError({ 'errors': serializer.errors})`